src/HL/View/Home/Features.hs

```diff
@@ -153,11 +145,10 @@ inference =
 lazy :: Html ()
 lazy =
   do h2_ "Lazy"
-     p_ "Functions don't evaluate their arguments. This means that programs \
-       \can compose together very well, with the ability to write control \
-       \constructs (such as if/else) just by writing normal functions. The purity \
-       \of Haskell code makes it easy to fuse chains of functions together, allowing \
-       \for performance benefits."
+     p_ "Values are computed when they are needed, rather than when they are defined in code. \
+        \ If a value you defined is not used, it never gets evaluated. This means you can spend \
+        \less time thinking about how your program is executed, and more time thinking about what \
+        \your program does."
```

Comment:
This write-up is still too long in my opinion. Can we make it even shorter? (Idea: delete the first sentence.)